docs/manual/nodes.md

```diff
@@ -57,6 +57,30 @@ We use `<repository>` image name throughout the docs articles. This means that t
 docker tag srlinux:20.6.1-286 srlinux:latest
 ```
 
+### subject alternative names
+
+With `subjectAltNames` the user sets the Subject Alternative Names that will be added to the node's certificate. Host names that are set by default are:
+
+- `shortName`
+- `longName`
+- `FQDN`
```

Comment:
let's have a real example. i.e. for a topology node named "srl1", the following SANs are set by default